app/Config/Autoload.php

```diff
@@ -31,12 +31,12 @@ class Autoload extends AutoloadConfig
      * else you will need to modify all of those classes for this to work.
      *
      * Prototype:
-     *```
+     * ```
```

Comment:
Let's just drop all of these. No need for markdown back-ticks in block comments in a PHP file 🤔
```suggestion
```

Reply:
Done.